app/controllers/partner_users_controller.rb

```diff
@@ -0,0 +1,93 @@
+# frozen_String_literal: true
+
+class PartnerUsersController < ApplicationController
+  before_action :set_partner, only: %i[index create destroy resend_invitation]
+
+  def index
+    @users = @partner.profile.users
```

Comment:
This is going to change with the merge PR :( Maybe rebase off that branch for this?